scalyr_agent/util.py

```diff
@@ -1626,6 +1779,12 @@ def acquire_token(self):
 
             # Head token is ripe.
             token = self._token_queue.popleft()
+            if self._logger:
+                import scalyr_agent.scalyr_logging as scalyr_logging
```

Comment:
I wonder if the rate limiter should be moved out of `util.py` (maybe `monitor_utils/blocking_rate_limiter.py`) then we can just have a single `import scalyr_agent.scalyr_logging` at the top rather than littered through the code.

Reply:
Good suggestion.  I've been thinking that we should make our modules smaller in general.  (When they get too long they also become harder to read)